content/docs/SUSHI/configuration/exhaustive-config.yaml

```diff
@@ -173,6 +193,18 @@ parameters:
   excludettl: true
   validation: [allow-any-extensions, no-broken-links]
   show-inherited-invariants: false
+# For R5 ImplementationGuides, the IG.definition.parameter.code became a Coding
+# In this case, the YAML key becomes the code.code. If the code provided is in the
+# value set bound in the IG definition (see: http://hl7.org/fhir/2022Sep/valueset-guide-parameter-code.html),
+# that system is automatically set. If no system is provided, SUSHI will default the system
+# to the system for the IG Parameter Codes
+# (see: http://build.fhir.org/ig/FHIR/fhir-tools-ig/branches/master/CodeSystem-ig-parameters.html).
+# To use a different system, the YAML key can be a FSH code.
+# An example parameters configuration for an R5 ImplementationGuide:
+# parameters:
+#   generate-xml: true
+#   validation: [allow-any-extensions, no-broken-links]
+#    http://example.org/fhir/other-system#example: false
```

Comment:
Oops. Too many spaces!
```suggestion
#   http://example.org/fhir/other-system#example: false
```

Reply:
I'd like to blame the shortcut for commenting lines, but I bet it was just my fault.